s3_to_redshift.go

```diff
@@ -99,8 +100,11 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	}
 
 	// COPY direct into it, ok to do since we're in a transaction
-	if err = db.JSONCopy(tx, inputConf, true, gzip); err != nil {
-		return fmt.Errorf("err running JSON copy: %s", err)
+	// can't switch on file ending as manifest files b/c
+	// manifest files obscure the underlying file types
+	// instead just pass the delimiter along even if it's null
+	if err = db.Copy(tx, inputConf, delimiter, true, gzip); err != nil {
```

Comment:
Nit, it would be nice if these error variables were scoped and not reused.
```
if err := db.Copy...
```
or even
```
if copyErr :=
```